clients/da-vinci-client/src/main/java/com/linkedin/davinci/consumer/StatefulVeniceChangelogConsumer.java

```diff
@@ -29,7 +30,7 @@
  * @param <K> Key type
  * @param <V> Value type
  */
-public interface BootstrappingVeniceChangelogConsumer<K, V> {
+public interface StatefulVeniceChangelogConsumer<K, V> {
```

Comment:
Since the client is relatively new and users are limited but just wanted to call out that this is a "breaking" change that will require users to fix their code once they bump their dependency.